aten/src/ATen/core/function_schema.h

```diff
@@ -509,6 +509,18 @@ inline std::ostream& operator<<(std::ostream& out, const Argument& arg) {
         unopt_type->kind() == c10::TypeKind::StringType) &&
         arg.default_value().value().isString()) {
       printQuotedString(out, arg.default_value().value().toStringRef());
+    } else if (type->kind() == TypeKind::ListType && type->castRaw<ListType>()->getElementType()->kind() == c10::TypeKind::IntType) {
+      // We want to faithfully replicate JIT schema.
+      // in native_functions.yaml defaults for int arrays always look like
+      //   int[2] stride=1
+      // instead of
+      //   int[2] stride=[1, 1]
+      auto default_val = arg.default_value().value().toIntList();
+      if (default_val.size() > 1) {
```

Comment:
Will there be cases that the items in `default_val` are different (e.g. [1, 3, 2] rather than [1, 1, 1])? Should we either check that or assert all items should be equal ?

Reply:
good point, ill guard for that case